src/org/jitsi/impl/neomedia/rtp/TimestampUtils.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright @ 2017 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.impl.neomedia.rtp;
+
+import org.jitsi.util.RTPUtils;
+
+/**
+ * Helper class to perform various timestamp manipulations and comparisons
+ * @author Ethan Lin
+ * //FIXME(brian): perhaps this should be moved alongside RTPUtils (or maybe its
+ * contents combined with RTPUtils?  This class deals with RTP timestamps)
```

Comment:
I wouldn't mind that. A larger scale re-organization is going to happen at some point in the future because, unfortunately, there are methods for RTP/RTCP stuff manipulation all over the place, i.e. `RawPacket`, `RTCPHeaderUtils`, and whatnot.